tensorboard/plugins/hparams/hparams_demo.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Write sample summary data for the hparams plugin.
+Each training-session here is a temperature simulation and records temperature
+related metric. See the function `run` below for more details.
+
+This demo is a slightly modified version of
+tensorboard/plugins/scalar/scalar_demo.py.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os.path
+import md5
```

Comment:
Pylint is complaining that this doesn't exist under python 3.  I think you probably want to import hashlib instead and use hashlib.md5().